pkg/oc/admin/prune/images.go

```diff
@@ -162,6 +165,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 	cmd.Flags().StringVar(&opts.CABundle, "certificate-authority", opts.CABundle, "The path to a certificate authority bundle to use when communicating with the managed Docker registries. Defaults to the certificate authority data from the current user's config file. It cannot be used together with --force-insecure.")
 	cmd.Flags().StringVar(&opts.RegistryUrlOverride, "registry-url", opts.RegistryUrlOverride, "The address to use when contacting the registry, instead of using the default value. This is useful if you can't resolve or reach the registry (e.g.; the default is a cluster-internal URL) but you do have an alternative route that works. Particular transport protocol can be enforced using '<scheme>://' prefix.")
 	cmd.Flags().BoolVar(&opts.ForceInsecure, "force-insecure", opts.ForceInsecure, "If true, allow an insecure connection to the docker registry that is hosted via HTTP or has an invalid HTTPS certificate. Whenever possible, use --certificate-authority instead of this dangerous option.")
+	cmd.Flags().BoolVar(opts.LeaveRegistryAlone, "leave-registry-alone", *opts.LeaveRegistryAlone, "If true, the prune operation will clean up image API objects in master and etcd, but none of the associated content in the registry is removed.")
```

Comment:
What about `--metadata-only`?  
`s/in master and etcd/in etcd/`
I'd add a recommendation to run this only in case of enormously large etcd/too many images and errors from pruning blobs.

Reply:
recommendation / string change part of next push

yeah, to elaborate on my thought process here, I considered things like `--soft-prune` and `--metadata-only` but felt like they were terminology that speaks to use and developers/owners of the code, but perhaps wouldn't to users with less familiarity / background

where as `--leave-registry-alone`, klunky as it might be speaks of the registry concept which the user should be familiar with.... I also considered `--do-not-prune-registry`.  At the time, I thought that  was not as good, but in hindsight, perhaps it is better than `--leave-registry-alone`.

@miminar @jim-minter @bparees thoughts?

Reply:
"If true, the prune operation will clean up image API objects in master and etcd, but none of the associated content in the registry is removed."

->

"If true, the prune operation will clean up image API objects, but the none of the associated content in the registry is removed."

As for the flag name, "--metadata-only", "--skip-registry", "--do-not-prune-registry", or "--skip-blobs" would all be preferred for me, in the end this flag is going to come down to how good the help+doc are(the flag name is never going to convey enough information), so i'd rather have a "clean" name than a clunky one.

Reply:
OK, I'll go with `--skip-registry`

Change of first sentence of help to @bparees recommendation part of next push

re: the recommendation from @miminar on when to run, my next push currently has:  
`This option make most sense when there is either a large amount of data in etcd and/or a large number of images or blobs.  You would then use this option to clean up etcd, and use the 'hard prune' administrative task (https://docs.openshift.org/latest/admin_guide/pruning_resources.html#hard-pruning-registry) to clean up the registry.`

Reply:
i'm not sure where you're intending for that text to appear, but i'd suggest it should really only be part of the documentation.  It's too wordy to convey in CLI help.  (Also direct links to the doc in CLI output are problematic because the OCP version of the CLI should send users to the OCP docs, not the origin docs (and in particular, to the specific OCP docs for their version).

We need a general image pruning recommendation/best-practices guide (should also cover *what* pruning does (what will be pruned under what conditions, what references are respected, etc) and this would belong there imho.

Reply:
sorry, I was proposing putting that text in the CLI help in response to @miminar 's suggestion

I'll remove and just stick to the first sentence.

Of course the best practice guide you mention sounds useful.  Based on the exchanges in the card, I had only planned on making the necessary (and smaller in scope) changes specific to the new option in the existing https://docs.openshift.org/latest/admin_guide/pruning_resources.html#pruning-images where a broader best practice guide would be beyond that scope.  That said, perhaps this new sentence would be part of that update.

Thoughts?